dll/win32/browseui/addresseditbox.cpp

```diff
@@ -311,23 +306,22 @@ HRESULT STDMETHODCALLTYPE CAddressEditBox::Execute(long paramC)
         return hr;
 
     /*
-     * Browsing to the pidl failed so it's not a folder. So invoke its defaule command.
+     * Browsing to the pidl failed so it's not a folder. So invoke its default command.
      */
-    HWND topLevelWindow;
-    hr = IUnknown_GetWindow(pisb, &topLevelWindow);
-    if (FAILED(hr))
-        return hr;
+    HWND hWnd;
+    if (FAILED(IUnknown_GetWindow(pisb, &hWnd)))
```

Comment:
Do you confirm that here we don't really care about having hr set to whether GetWindow succeeds or not?